modules/programs/foot.nix

```diff
@@ -0,0 +1,75 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.foot;
+  iniFormat = pkgs.formats.ini { };
+in {
+  meta.maintainers = with lib.maintainers; [ plabadens ];
+
+  options.programs.foot = {
+    enable = mkEnableOption "Foot terminal";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.foot;
+      defaultText = literalExample "pkgs.foot";
+      description = "The foot package to install";
+    };
+
+    server.enable = mkEnableOption "Foot terminal server";
+
+    settings = mkOption {
+      type = iniFormat.type;
+      default = { };
+      description = ''
+        Configuration written to
+        <filename>~/.config/foot/foot.ini</filename>. See <link
```

Comment:
```suggestion
        <filename>$XDG_CONFIG_HOME/foot/foot.ini</filename>. See <link

Reply:
Yes, that's more accurate, although it seems existing modules don't always do this.